jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -216,11 +216,23 @@
 		}
 		
 		Lookup(Class<?> lookupClass, Class<?> prevLookupClass, int lookupMode) {
-			this(lookupClass, prevLookupClass, lookupMode, true);
+			this(lookupClass, prevLookupClass, lookupMode,
+			/*[IF Java15]
+				false
+			/*[ELSE] Java15*/
+				true
+			/*[ENDIF] Java15*/
+			);
 		}
 		
 		Lookup(Class<?> lookupClass, int lookupMode, boolean doCheck) {
-			this(lookupClass, null, lookupMode, doCheck);
+			this(lookupClass, null, lookupMode,
+			/*[IF Java15]
+				false
+			/*[ELSE] Java15*/
+				doCheck
+			/*[ENDIF] Java15*/
```

Comment:
This looks wrong - code that's explicitly asking for the check should be updated for 15 rather than discarding the value here.

Reply:
The approach of this PR is to always skip the package check other than those three APIs in question for `JDK15`. Those three have been modified to invoke the `Lookup` constructor with `true` flag for `JDK15` (to match RI behaviours). The expectation is that all other JDK15 API don't perform such check.
Does this make sense?